### PR TITLE
Move includes/utils to services

### PR DESCRIPTION
**Pull request by jitendrapurohit**
Overview
----------------------------------------
Move utils.inc to services

Before
----------------------------------------
.inc file

After
----------------------------------------
- New d8 service added webform_civicrm.utils
- Direct function call replaced to use services.

Comments
----------------------------------------
ping @KarinG

### CODE REVIEW

**Comment by KarinG** [comment]
Re: tests -> we're hitting this -> 

`Drupal\Tests\webform_civicrm\FunctionalJavascript\ActivitySubmissionTest::testSubmitWebform
Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: You have requested a non-existent service "url_generator".`

It's not immediately clear to me what this PR changes in `tests/src/FunctionalJavascript/ActivitySubmissionTest.php` that would cause this error but suspect it's in here. Why add a new Container in (all) tests?

```
    $utils = new Utils();
    $container = new ContainerBuilder();
    \Drupal::setContainer($container);
    $container->set('webform_civicrm.utils', $utils);
```

**Comment by jitendrapurohit** [comment]
Ah yes thanks @KarinG, seems the service container initialization was only required on the utils test and not on the functional-javascript ones. Fixed. All green now :).

File: tests/src/FunctionalJavascript/ContactSubmissionTest.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 namespace Drupal\Tests\webform_civicrm\FunctionalJavascript;
 
 use Drupal\Core\Url;
+use Drupal\webform_civicrm\Utils;
```
Comment (KarinG) [inline tests/src/FunctionalJavascript/ContactSubmissionTest.php]: Do we need the `use` here? I don't see it in `ActivitySubmissionTest.php` e.g.

**Comment by KarinG** [comment]
Wow big PR! I looked at it noted some inconsistencies in the use of `use` 🙂 👆 If you could please have one more look to see if there can be some more tidying up that would be great.

File: tests/src/FunctionalJavascript/ContactSubmissionTest.php (hunk 1 above)
Comment (jitendrapurohit) [reply tests/src/FunctionalJavascript/ContactSubmissionTest.php]: Nope, it wasn't required. I've removed it. Thanks for pointing that 👍

**Comment by KarinG** [comment]
Ok merging this! That was a big one 👍

**Pull request merged**